core/graph/src/test/java/uk/gov/gchq/gaffer/integration/operation/named/cache/NamedOperationCacheIT.java

```diff
@@ -68,18 +68,20 @@ public class NamedOperationCacheIT {
 
     private User user = new User();
     private User authorisedUser = new User.Builder().userId("authorisedUser").opAuth("authorised").build();
-    private User adminAuthUser = new User.Builder().userId("adminAuthUser").opAuth(adminAuth).build();
+    private User adminAuthUser = new User.Builder().userId("adminAuthUser").opAuth(ADMIN_AUTH).build();
     private Context context = new Context(user);
     private GetAllNamedOperationsHandler getAllNamedOperationsHandler = new GetAllNamedOperationsHandler(SUFFIX);
     private AddNamedOperationHandler addNamedOperationHandler = new AddNamedOperationHandler(SUFFIX, true);
     private GetAllNamedOperationsHandler getAllNamedOperationsHandler1 = new GetAllNamedOperationsHandler(SUFFIX);
     private DeleteNamedOperationHandler deleteNamedOperationHandler = new DeleteNamedOperationHandler(SUFFIX);
     private GetAllNamedOperations get = new GetAllNamedOperations();
```

Comment:
It might also be worth removing the GetAllNamedOperations get = new GetAllNamedOperations() line in each test as there is already this same line here so it is needless repetition ⁠

Reply:
@cn337131 
Thank you for your helpful reviews. I have addressed your comments by removing the duplicated GetAllNamedOperations creation code and using the STORE mock variable directly. Please let me know if there are any other improvements I can make. I appreciate you taking the time to review my changes.